### PR TITLE
aarch64 support added

### PR DESCRIPTION
**Pull request by moorthy-bs**
Signed-off-by: Moorthy Baskar <moorthy.baskaravenkatraman-sambamoorthy@linaro.org>

### CODE REVIEW

**Comment by moorthy-bs** [comment]
`__aarch64__` is the correct predefined macro for ARM64. Ref. [http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0774g/chr1383660321827.html](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0774g/chr1383660321827.html)

File: src/rtRemoteValueWriter.cpp [hunk 1]
```diff
@@ -138,7 +138,7 @@ rtRemoteValueWriter::write(rtRemoteEnvironment* env, rtValue const& from,
     case RT_uint64_tType: to.AddMember("value", from.toUInt64(), doc.GetAllocator()); break;
     case RT_stringType:   to.AddMember("value", std::string(from.toString().cString()), doc.GetAllocator()); break;
     case RT_voidPtrType:
-#if __x86_64
+#if __x86_64 || __aarch64__
```
Comment (psivasubramanian) [inline src/rtRemoteValueWriter.cpp]: This is suffice for aarch64 build, others are good to have to avoid werrors.

**Comment by psivasubramanian** [comment]
Is there anything pending in this PR to get merged?

**Comment by moorthy-bs** [comment]
Hi @gladish , is there any more approval for this PR?

**Comment by moorthy-bs** [comment]
Hi @gladish , could you please point someone to approve/merge this PR?

**Comment by moorthy-bs** [comment]
Hi @gladish , is anything blocking to merge this PR?

**Comment by moorthy-bs** [comment]
Hi @gladish , rebased my fork to latest master branch. I think this can be merged if there is no blocking

**Comment by moorthy-bs** [comment]
Hi @mfiess , this is an approved PR. Please take a look.

**Review by mfiess — APPROVED** [review]

**Pull request merged**